### PR TITLE
Seeds file now creates default collaborators for OpenStax and Rice

### PR DESCRIPTION
**Pull request by Dantemss**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -120,4 +120,15 @@
   license.allows_derivatives    = true
 end
 
+# Output logging info to stdout
+original_logger = Rails.logger
```
Comment (jpslav) [inline db/seeds.rb]: Backburner is that we need to make a temp STDOUT logger block in OX utilities:

```ruby
Stdout.logging do
  blah
end
```

File: spec/routines/create_default_collaborators_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    result = nil
+    expect{
+      result = CreateDefaultCollaborators.call
+    }.to change{ User.count }.by(2)
+    expect(result.errors).to be_empty
+
+    expect(result.outputs.author.username).to eq 'openstax'
+    expect(result.outputs.author.full_name).to eq 'OpenStax'
+
+    expect(result.outputs.copyright_holder.username).to eq 'rice_university'
```
Comment (jpslav) [inline spec/routines/create_default_collaborators_spec.rb]: Can we drop the underscore?

**Comment by jpslav** [comment]
One minor comment, then merge.

File: spec/routines/create_default_collaborators_spec.rb (hunk 2 above)
Comment (Dantemss) [inline spec/routines/create_default_collaborators_spec.rb]: Sure.

**Pull request merged**